package java8_3.ch08_exception.try_catch;

public class ExceptionEx9 {
    public static void main(String[] args) {
        throw new RuntimeException();
    }
}
